README.md

```diff
@@ -68,7 +75,11 @@ To initialize a ForageSDK instance, you need to provide the environment.
 
 ```swift
 ForageSDK.setup(
-    ForageSDK.Config(environment: .sandbox)
+    ForageSDK.Config(
+        environment: .sandbox,
+        merchantID: "mid/abcd123",
```

Comment:
Let's use the old pattern here still. `merchantID: "1234567"`

Reply:
New SDK users are instructed to use the "Merchant ID" across our docs, which has this format. Are we not fully set on this format or am I missing some other context?

Reply:
That was changed to return to the original format. We aren't pushing that format until we have food as medicine customers.